src/pages/Vaults/Vault/components/IssueRedeemForm/IssueRedeemForm.tsx

```diff
@@ -187,7 +191,7 @@ const IssueRedeemForm = ({
             <StyledInputLabel id={amountLabelId}>{label}</StyledInputLabel>
             <TokenInput
               aria-labelledby={amountLabelId}
-              tokenSymbol='BTC'
+              token='BTC'
```

Comment:
Should this be ticker? We've been quite inconsistent on this, but I've noticed that you've used a `ticker` prop in several places recently, and I think it makes sense.

Reply:
Good point